lib/canary/plugs.ex

```diff
@@ -181,6 +181,27 @@ defmodule Canary.Plugs do
   plug :load_resource, model: Post, id_name: "slug", id_field: "slug", only: [:show], persisted: true
   ```
   """
+  def authorize_controller(%{assigns: %{current_user: nil}} = conn, opts) do
```

Comment:
I created this based on your current test.
So in case there is no "current_user" is assigned your tests fails. so I added this for the test to fail as well